env.mk.sample

```diff
@@ -14,20 +14,12 @@ DOMAIN=my-zone.cloud-tutorial.dev
 GIT_USER_ID=GoogleCloudPlatform
 GIT_REPO_ID=cloud-run-anthos-reference-web-app
 
-# Cluster information
-CLUSTER_LOCATION=$(shell gcloud --project $(PROJECT_ID) container clusters list --filter="name:$(CLUSTER_NAME)" --format="csv[no-heading](location)" )
-CLUSTER_GKE_VERSION=$(shell gcloud --project $(PROJECT_ID) container clusters list --filter="name:$(CLUSTER_NAME)" --format="csv[no-heading](currentMasterVersion)")
-
-# Default location and GKE version for cluster creation
-ifeq ($(CLUSTER_LOCATION),)
-	CLUSTER_LOCATION=us-west1-a
-endif
-ifeq ($(CLUSTER_GKE_VERSION),)
-	CLUSTER_GKE_VERSION=1.15
-endif
+# Cluster information. Uses defaults if cluster does not exist
+CLUSTER_LOCATION=$(or $(shell gcloud --project $(PROJECT_ID) container clusters list --filter="name:$(CLUSTER_NAME)" --format="value(location)" ), us-west1-a)
+CLUSTER_GKE_VERSION=$(or $(shell gcloud --project $(PROJECT_ID) container clusters list --filter="name:$(CLUSTER_NAME)" --format="value(currentMasterVersion)"), 1.15)
```

Comment:
This style allows this to be evaluated only when the makefile variables are actually used.